doc/source/examples/sharp-immersed-boundary/3d-rbf-static-mixer/3d-rbf-static-mixer.rst

```diff
@@ -0,0 +1,293 @@
+=====================================================================================
+3D Static Mixer Using RBF Sharp-Immersed Boundary
+=====================================================================================
+
+The usage of `static mixers <https://en.wikipedia.org/wiki/Static_mixer>`_ is common for the blending of multi-component products in polymerization, as well as water treatment and chemical processing. They involve no moving part, but obtain mixing effects by converting pressure to radial mixing and flow division as the flow progresses through obstacles. Their advantages include low maintenance, no external energy input, continuous operation, compact design (pipe insertion), and easy installation.
+
+Because it is not possible to modulate the rotation speed or change the impeller, as is the case in stirred tanks, their design must be done precisely and take into account the desired level of mixing and the properties of fluids involved.
+
++-----------------------------------------------------------------------------------------------------------------------------+
+|  .. figure:: images/static_mixer_stl_casing_arrows.png                                                                      |
+|     :align: center                                                                                                          |
+|     :width: 800                                                                                                             |
+|     :name: Surface grid representation of a helix static mixer with its casing.                                             |
```

Comment:
I don't think we usually use captions for images in the examples of the documentation... it is nice, but I would remove it for the sake of having a uniform format :)